modules/gdscript/tests/scripts/runtime/features/lambda_bind_argument_count.gd

```diff
@@ -0,0 +1,52 @@
+# https://github.com/godotengine/godot/issues/93952
+
+func test_call():
+	pass
+
+func test_call_with_param(_arg):
+	pass
+
+func test():
+	# Lambdas without self.
+	var test_lambda_1 = func():
+		test_call()
```

Comment:
I think it's a self lambda since it calls an instance method.

https://github.com/godotengine/godot/blob/20ba2f00bd9199b675176a8e1ac151f96bfb5cfa/modules/gdscript/gdscript_analyzer.cpp#L3431-L3433

Reply:
It is but the arguments are not checked for this one, will move it out to make it disconnected